Changelog.md

```diff
@@ -1,5 +1,71 @@
 # Changelog
 
+### 1.0.1
+
+**Image:**
+TODO 
+
+**Known Issues**
+TODO 
```

Comment:
Please add here that Ingress now mandates cluster scoped access to IngressClass and this is being dealt in https://github.com/kubernetes/ingress-nginx/pull/7578

Reply:
done